docs/manifest.json

```diff
@@ -344,6 +343,21 @@
       ]
     ]
   ],
+  [
+    {
+      "title": "Security",
+      "icon": "lock-closed",
+      "root": "security"
+    },
+    [
+      ["Overview", "/security/overview"],
+      ["Vulnerability disclosure policy", "/security/vulnerability-disclosure-policy"],
+      ["XSS leak protection", "/security/xss-leak-protection"],
+      ["CSRF protection", "/security/csrf-protection"],
+      ["Fixation protection", "/security/fixation-protection"],
+      ["Password protection and rules", "/security/password-protection"]
```

Comment:
```suggestion
      ["Password protection and rules", "/security/password-protection-and-rules"]
```

Reply:
actually, I changed the file name bc there are links within our docs that point to /docs/security/password-protection 
did you want to me to change the file name back? and then hunt down those links? it's possible they were in sanity and may need redirects - I don't remember

Reply:
Oh, missed that 🤦

We can rename as you have done here! I don't think there was a reason to change it so better to stick with the old one. Can you update the spreadsheet as well to make sure we don't redirect to the new (old) location?